jax/_src/cudnn/fused_attention_stablehlo.py

```diff
@@ -986,8 +979,8 @@ def dot_product_attention(query: Array,
   """
   # check if cuDNN is installed
   cudnn_version = check_cudnn_version()
-  # only support Ampere and Hopper for now
-  check_compute_capability((80, 90))
+  # only support at least Ampere
+  jtu.is_cuda_compute_capability_at_least("8.0")
```

Comment:
`jtu` is for testing. Perhaps inline it here instead?

Note also that atm you aren't asserting anything.